action.yml

```diff
@@ -0,0 +1,17 @@
+name: 'Apply patch'
+description: 'Apply patch is created for applying a patch generated from Github compare commits'
```

Comment:
Instead of saying what is was created for, it's better if this says what it does